app/assets/stylesheets/application.scss

```diff
@@ -1,73 +1,37 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require datatables.net-bs/css/dataTables.bootstrap.css
- *= require datatables.net-fixedheader-bs/css/fixedHeader.bootstrap.css
- *= require_tree .
- *= require_self
- */
-$maroon: #881c1c;
-$slightly-maroon: #eacccc;
-
-$body-bg: $maroon;
 $font-family-sans-serif: verdana, arial, helvetica, sans-serif;
```

Comment:
So,  Do we think this font family is important?

[This is Bootstrap's default sans-serif font stack](https://getbootstrap.com/docs/4.3/content/reboot/#native-font-stack)

Reply:
And [This is what UMass "wants"](https://www.umass.edu/brand/elements/typography#webfonts)

Reply:
I can't remember any reason for that. Do what you will.